samples/maps/demo/animated-map/demo.js

```diff
@@ -0,0 +1,231 @@
+Highcharts.getJSON(
+    "https://code.highcharts.com/mapdata/custom/world.topo.json",
+    topology => {
+    // Create a data value for each feature
+        const data = [];
+        data.push({
```

Comment:
Instead of initializing and empty array then pushing, just create an array with object literals.